### PR TITLE
Updated company schema

### PR DESCRIPTION
**Pull request by motasimmakki**

### CODE REVIEW

File: companies/apple.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "name": "Apple Distribution International",
     "address": "Hollyhill Industrial Estate\nHollyhill, Cork\nIreland",
     "email": "contactus.de@euro.apple.com",
+    "phone": "+353 21 428 4000",
```
Comment (baltpeter) [inline companies/apple.json:12]: Please add sources for all points of contact listed in the record.

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks for the record! I have requested some changes below.

File: companies/apple.json (hunk 1 above)
Comment (motasimmakki) [reply companies/apple.json:12]: @baltpeter I've added the source of contact, thanks for mentioning it.

File: companies/apple.json [hunk 2]
```diff
@@ -9,10 +9,12 @@
...
     "web": "https://www.apple.com",
     "sources": [
         "https://www.apple.com/legal/privacy/de-ww/",
-        "https://www.apple.com/de/contact/"
+        "https://www.apple.com/de/contact/",
+        "https://britainfirm.com/ireland/apple-distribution-international-542875"
```
Comment (baltpeter) [inline companies/apple.json:17]: This unfortunately seems like quite a spammy and unreliable site. We would need one preferably from the official Apple site or, barring that, from a trusted third-party site.

**Review by baltpeter — CHANGES_REQUESTED** [review]

File: companies/apple.json (hunk 2 above)
Comment (motasimmakki) [reply companies/apple.json]: Thanks for your concern, now I added the verified contacts from the official site of Apple with the proper source.

**Review by baltpeter — CHANGES_REQUESTED** [review]
As you can see from the test failure, this doesn't conform to our schema.

You can also run the tests locally: https://github.com/datenanfragen/data#testing-records

**Comment by motasimmakki** [comment]
> As you can see from the test failure, this doesn't conform to our schema.
> 
> You can also run the tests locally: https://github.com/datenanfragen/data#testing-records

Fixed it, now it looks fine.

**Comment by baltpeter** [comment]
According to the source you linked, the phone number is for purchasing over the phone, not for data protection requests. I don't think there is such a phone number for Apple.

**Pull request closed without merging**